qlib/workflow/recorder.py

```diff
@@ -220,14 +220,45 @@ def __init__(self, experiment_id, uri, name=None, mlflow_run=None):
                 else None
             )
 
+    def __repr__(self):
+        name = self.__class__.__name__
+        space_length = len(name) + 1
+        return "{name}(info={info},\n{space}uri={uri},\n{space}artifact_uri={artifact_uri},\n{space}client={client})".format(
+            name=name,
+            space=" " * space_length,
+            info=self.info,
+            uri=self.uri,
+            artifact_uri=self.artifact_uri,
+            client=self.client,
+        )
+
+    @property
+    def uri(self):
+        return self._uri
+
+    @property
+    def artifact_uri(self):
+        return self._artifact_uri
+
+    @property
+    def root_uri(self):

```

Comment:
What is this method for? Can you give us more explanation about it?

Reply:
This is to return the path of a specific record. For example, you can call `R.get_recorder().root_uri` and it will return: `ml-runs/2/32792162f0f44951b1285c0b2e489252`.

Reply:
Please update the related `docs`.